### DISCUSSION TITLE
Powerwall 3 aka Powerwall++ (1707000-xx-y)

### DISCUSSION THREAD

**Discussion by jasonacox** [jasonacox/Powerwall-Dashboard#352] [General]
Just starting to hear about Powerwall 3...  I would love to hear from anyone who gets one as is able to use the Dashboard with the PW3.

https://twitter.com/elonmusk/status/1700038677404242106?s=20

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/836718/1d78113d-bbb6-4091-b779-da6b52f19497)

Details from:  https://teslamotorsclub.com/tmc/threads/discussion-powerwall-3-speculation-discussion-etc.299492/page-8#post-7806554

* Size is approx. 43"x24"x7".
* The circuit breaker in the Backup Gateway 2 which connects to the PW3 is in fact 60A.
* Looks like it is 13.5 kWh ([here](https://teslamotorsclub.com/tmc/threads/discussion-powerwall-3-speculation-discussion-etc.299492/page-9)) - Same as Powerwall 2 and Powerwall+, so it is basically just an improved version. Higher output power and likely LFP chemistry.
* Weight is 287 lbs, which is in between Powerwall+ at 344 lbs and Powerwall 2 at 251 lbs.
* Appears to be called Powerwall++ ([from manual here](https://teslamotorsclub.com/tmc/attachments/powerwall-3-manual-pdf.972289/))

**Comment by MattMake** [comment]
Definitely lacking cooling vents, another indicator of possible LFP chemistry.  (I doubt they would vent through the top for obvious reasons).  Unless intake and exhaust are all on the other side.

Not sure I like where this one was placed, for a glass front, that's awfully close to the ground for rocks getting kicked or tossed around.

**Reply by longzheng** [reply]
There are still cooling vents at the bottom and back.

<img width="880" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/484912/704455d2-7e32-4287-978e-63e2bb16dc01">
